article/app/renderers/RemoteRenderer.scala

```diff
@@ -93,10 +94,11 @@ class RemoteRenderer {
     ws:WSClient,
     path: String,
     page: PageWithStoryPackage,
-    blocks: Blocks
+    blocks: Blocks,
+    context: ApplicationContext
```

Comment:
Yeah, it's not clear what ApplicationContext is (the name is unfortunately vague) or why it is needed here. Might be better to pull out the exact arguments required and pass them explicitly, or do as @MatthewJWalls suggests and construct the commercial object separately and then pass in.

Reply:
Will do!